lib/portmidi/input/reader.ex

```diff
@@ -35,12 +37,15 @@ defmodule PortMidi.Input.Reader do
 
   defp loop(server, stream) do
     if do_poll(stream) == :read, do: read_and_send(server,stream)
+    :timer.sleep(@input_poll_sleep)
     loop(server, stream)
   end
 
   defp read_and_send(server, stream) do
-    messages = do_read(stream, @buffer_size)
-    Server.new_messages(server, messages)
+    case do_read(stream, @buffer_size) do
+      {:error, reason} -> Logger.debug("Error Reading Midi: #{reason}")
```

Comment:
Just a thought - it would be nice to design a way to let the caller specify the behaviour when such a message happens (e.g. a configurable callback of some sort), maybe with a default implementation?